extensions/amp-image-slider/0.1/test/integration/test-amp-image-slider.js

```diff
@@ -123,6 +129,10 @@ config.run('amp-image-slider', function() {
           .then(prepareSlider);
     }
 
+    function timeout(win, ms) {
```

Comment:
`Services.timerFor(win).promise(ms)`

Reply:
I am not quite sure if `Services` is safely accessible in integration tests. For normal tests it should work. Will double check

Reply:
Just tried, the timer service is not available in compiled integration test. Probably would just go with `setTimeout` (otherwise need to go through the registration process)